bin/amazon_ssa_agent

```diff
@@ -4,63 +4,8 @@ Dir.chdir(root_dir) { require 'bundler/setup' }
 $LOAD_PATH << File.join(root_dir, "lib")
 
 require 'optparse'
-require 'log4r'
-require 'log4r/configurator'
 require 'log_decorator'
-
 require 'aws-sdk'
-
-#
-# We must do this before anything accesses log4r.
-#
-module Log4r
-  Configurator.custom_levels(:DEBUG, :INFO, :WARN, :ERROR, :FATAL, :COPY)
-end
-
-class LogFormatter < Log4r::Formatter
-  @prog         = File.basename(__FILE__, ".*")
-  @extractor_id = ''
-  def format(event)
-    return event.data.chomp + "\n" if event.level == Log4r::COPY
-
-    "#{Log4r::LNAMES[event.level]} [#{datetime}" +
-      (@extractor_id.nil? ? "" : " #{@extractor_id}") +
-      "] -- #{@prog}: " +
-      (event.data.kind_of?(String) ? event.data : event.data.inspect) + "\n"
-  end
-
-  def self.extractor_id=(val)
-    @extractor_id = val
-  end
-
-  private
-
-  def datetime
-    time = Time.now.utc
-    time.strftime("%Y-%m-%dT%H:%M:%S.") << "%06d" % time.usec
-  end
-end
-
-LOG_LEVELS = {
-  'DEBUG'      => Log4r::DEBUG,
-  'INFO'       => Log4r::INFO,
-  'WARN'       => Log4r::WARN,
-  'ERROR'      => Log4r::ERROR,
-  'FATAL'      => Log4r::FATAL,
-  'COPY'       => Log4r::COPY,
-  Log4r::DEBUG => "DEBUG",
-  Log4r::INFO  => "INFO",
-  Log4r::WARN  => "WARN",
-  Log4r::ERROR => "ERROR",
-  Log4r::FATAL => "FATAL",
-  Log4r::COPY  => "COPY"
-}
-
-::Module.include(LogDecorator::Logging::ClassMethods)
-include LogDecorator::Logging
-$log = Log4r::Logger.new('toplog')
-LogDecorator.logger = $log
```

Comment:
@Fryguy There are some errors when I run a scanning test.  To use _log calls, we have to assign $log to LogDecorator.logger.